metric/aggregated/types.go

```diff
@@ -31,25 +31,25 @@ import (
 // Metric is a metric, which is essentially a named value at certain time.
 type Metric struct {
 	metric.ID
-	Timestamp time.Time
-	Value     float64
+	TimeNs int64
```

Comment:
I think Unix though would be descriptive in that infers that it measures time since epoch is 1970 Jan 1.  TimeNanos is less descriptive IMO.

I'm not bullish on either but we should standardize on one or the other across projects.